2020:04:01 06:07:26:458547 ~ Traceback (most recent call last):
  File "C:\SDK\Anaconda3\lib\site-packages\pydebugger\debug.py", line 222, in read_config
    data = self.cfg.get(section, option)
  File "C:\SDK\Anaconda3\lib\configparser.py", line 780, in get
    d = self._unify_values(section, vars)
  File "C:\SDK\Anaconda3\lib\configparser.py", line 1146, in _unify_values
    raise NoSectionError(section) from None
configparser.NoSectionError: No section: 'DEBUG'

2020:04:01 06:08:40:153351 ~ Traceback (most recent call last):
  File "C:\SDK\Anaconda3\lib\site-packages\pydebugger\debug.py", line 222, in read_config
    data = self.cfg.get(section, option)
  File "C:\SDK\Anaconda3\lib\configparser.py", line 780, in get
    d = self._unify_values(section, vars)
  File "C:\SDK\Anaconda3\lib\configparser.py", line 1146, in _unify_values
    raise NoSectionError(section) from None
configparser.NoSectionError: No section: 'COLORS'

2020:04:01 06:08:40:649362 ~ Traceback (most recent call last):
  File "C:\SDK\Anaconda3\lib\site-packages\pydebugger\debug.py", line 222, in read_config
    data = self.cfg.get(section, option)
  File "C:\SDK\Anaconda3\lib\configparser.py", line 780, in get
    d = self._unify_values(section, vars)
  File "C:\SDK\Anaconda3\lib\configparser.py", line 1146, in _unify_values
    raise NoSectionError(section) from None
configparser.NoSectionError: No section: 'DEBUG'

2020:04:01 06:08:41:523560 ~ Traceback (most recent call last):
  File "D:\TOOLS\pyx\gitdate.py", line 218, in notify
    import PySnarl
ModuleNotFoundError: No module named 'PySnarl'

2020:04:01 06:08:44:229274 ~ Traceback (most recent call last):
  File "D:\TOOLS\pyx\gitdate.py", line 218, in notify
    import PySnarl
ModuleNotFoundError: No module named 'PySnarl'

